test/no-ignored-test-files.js

```diff
@@ -68,6 +68,11 @@ test('without AVA config in package.json', () => {
 				code: code(true),
 				filename: toPath('lib/foo.spec.js'),
 				options: [{files: ['lib/**/*.spec.js']}]
+			},
+			{
+				code: code(true),
+				filename: '<text>',
```

Comment:
Asked on the Gitter chat, apparently there is no good way to emulate `stdin`, so this is the best I could find.